src/Storages/MergeTree/KeyCondition.cpp

```diff
@@ -730,15 +735,15 @@ static NameSet getAllSubexpressionNames(const ExpressionActions & key_expr)
 
 void KeyCondition::getAllSpaceFillingCurves()
 {
-    /// So far the only supported function is mortonEncode (Morton curve).
```

Comment:
<img width="794" alt="image" src="https://github.com/ClickHouse/ClickHouse/assets/114298166/001175d1-4b37-4dfe-9486-145737067fd6">

There is a lot of space-filling functions, so I think it's better to keep this comment and add Hilbert curve there